### ISSUE TITLE
Error in first "Getting Started" example: need to install tensorflow_probability manually

### ISSUE THREAD

**Issue by rodrigodesalvobraz** [ray-project/ray#32966]
### What happened + What you expected to happen

I followed the instructions on the "[Getting Started with RLlib](https://docs.ray.io/en/latest/rllib/rllib-training.html#)" page to to run

```
pip install "ray[rllib]" tensorflow
rllib train --algo DQN --env CartPole-v1
```

however the latter failed with a Python message about being unable to import module tensorflow_probability.

It did work after I ran `pip install tensorflow_probability`, but of course the initial installation should have taken care of that.

### Versions / Dependencies

Ray on PyPI as of Mar 1st 2023, fresh conda environment with Python 3.8 on Windows 10 Enterprise.

### Reproduction script

```
pip install "ray[rllib]" tensorflow
rllib train --algo DQN --env CartPole-v1
```

### Issue Severity

Low: It annoys or frustrates me.

**Comment by zongtaili** [comment]
I comment this line of code `# import tensorflow_probability as tfp` in `“your path”\Lib\site-packages\ray\rllib\algorithms\bandit\bandit_tf_model.py`， it looks like work well，but I don’t know such operation whether make impact in the future

**Comment by Rohan138** [comment]
Will be fixed by #34756
